src/driver.rs

```diff
@@ -60,10 +61,58 @@ fn test_arg_value() {
 }
 
 #[allow(clippy::too_many_lines)]
+
+struct ClippyCallbacks;
+
+impl rustc_driver::Callbacks for ClippyCallbacks {
+    fn after_parsing(&mut self, compiler: &interface::Compiler<'_>) -> bool {
```

Comment:
```suggestion
    fn after_parsing(&mut self, compiler: &interface::Compiler) -> bool {
```

Won't compile again otherwise

Reply:
I'll just fix this myself and merge

Reply:
Fixed =P